python/snewpy/snowglobes_interface.py

```diff
@@ -113,8 +117,9 @@ def _load_channels(self, chan_dir):
                 # Format 2: explicit binning, differs per channel
                 tokens = l.split()[6:]
                 df = pd.read_table(f, delim_whitespace=True, index_col=1, comment='%', names=['name','n','parity','flavor','weight'], usecols=range(1,6))
-                # drop coherent scattering channels, which have different binning
-                df = df[df["name"].str.startswith('coh_') == False]
+                # drop coherent scattering channels if channel is argon_400bins, mixing channels which have different binning
```

Comment:
The comment is slightly misleading
```suggestion
                # drop coherent scattering channels if material is argon_400bins, mixing channels which have different binning
```
But do I understand correctly, that we skip some channels because in our data structure we have only one binning for each material, while Snowglobes can have separate binning for each channel? 
Probably this is something we should fix on our side.

Reply:
Good question! But either way, it’s certainly beyond the scope of this PR, so I moved this to issue #334 and added my thoughts there.

Reply:
Yes, indeed I opened an issue in snowglobes: 

https://github.com/SNOwGLoBES/snowglobes/issues/24

But as Jost said indeed opnining it in snewpy seems more reasonable, and that was goes beyond the scope of this PR.